_includes/releases/v21.2/v21.2.8.md

```diff
@@ -0,0 +1,101 @@
+- Fixed a bug when adding a [`hash-sharded index`](../v21.2/hash-sharded-indexes.html) to a table watched by a changefeed. [#77739][#77739]
+- Fixed successive schema change backfills from skipping spans that were checkpointed by an initial backfill that was restarted. [#77829][#77829]
+- Attempting to run concurrent profiles now works up to a concurrency limit of two. This will remove the occurrence of `profile id not found` errors while running up to two profiles concurrently. When a profile is not found, the error message has been updated to suggest troubleshooting steps. [#77977][#77977]
+- Fixed an optimizer bug that prevented expressions of the form `(NULL::STRING[] <@ ARRAY['x'])` from being folded to `NULL`. This bug was introduced in v21.2.0. [#78039][#78039]
+- Added a limit of seven concurrent asynchronous consistency checks per store, with an upper timeout of one hour. This prevents abandoned consistency checks from building up in some circumstances, which could lead to increasing disk usage as they held onto Pebble snapshots. [#77611][#77611]
+- Fixed a bug where the [**Statement Details**](../v21.2/ui-statements-page.html#statement-details-page) page fails to load query plan even after when the plan has been sampled. [#78105][#78105]
+- Fixed a memory leak in the Pebble block cache. [#78257][#78257]
+- Fixed a bug that caused internal errors when `COALESCE` and `IF` expressions had inner expressions with different types that could not be cast to a common type. [#78342][#78342]
+- CockroachDB might now fetch fewer rows when performing lookup and index joins on queries with the [`LIMIT`](../v21.2/limit-offset.html#limit) clause. [#78474][#78474]
+- A zone config change event now includes the correct details of what was changed instead of incorrectly displaying "undefined". [#78634][#78634]
+- Fixed a bug that prevented a table created on a 22.1 node to be queried on a 21.2 node in a mixed-version cluster. [#78657][#78657]
+- Fixed a bug that caused errors when trying to evaluate queries with `NULL` values annotated as a tuple type, such as `NULL:::RECORD`. This bug was present since version 19.1. [#78635][#78635]
+- Fixed a bug where CockroachDB could lose `INT2VECTOR` and `OIDVECTOR` types of some arrays. [#78630][#78630]
+- Fixed a bug that caused the optimizer to generate invalid query plans which could result in incorrect query results. The bug, which has been present since version 21.1.0, can appear if all of the following conditions are true: 1) the query contains a semi-join, such as queries in the form: `SELECT * FROM t1 WHERE EXISTS (SELECT * FROM t2 WHERE t1.a = t2.a);`, 2) the inner table has an index containing the equality column, like `t2.a` in the example query, 3) the index contains one or more columns that prefix the equality column, and 4) the prefix columns are `NOT NULL` and are constrained to a set of constant values via a `CHECK` constraint or an `IN` condition in the filter. [#78975][#78975]
```

Comment:
```suggestion
- Fixed a bug that caused the optimizer to generate invalid query plans which could result in incorrect query results. The bug, which has been present since v21.1.0, can appear if all of the following conditions are true: 1) the query contains a semi-join, such as queries in the form: `SELECT * FROM t1 WHERE EXISTS (SELECT * FROM t2 WHERE t1.a = t2.a);`, 2) the inner table has an index containing the equality column, like `t2.a` in the example query, 3) the index contains one or more columns that prefix the equality column, and 4) the prefix columns are `NOT NULL` and are constrained to a set of constant values via a `CHECK` constraint or an `IN` condition in the filter. [#78975][#78975]
```